courses/fundamentals_of_ada/labs/solar_system/18_multiple_inheritance/.~lock.questions.doc#

```diff
@@ -0,0 +1 @@
+,leo,leo-Latitude-7410,03.12.2020 19:57,file:///home/leo/.config/libreoffice/4;
```

Comment:
this file doesn't belong in the repository

Reply:
Yes, removing and adding it to gitignore